### DISCUSSION TITLE
Access regression parameters

### DISCUSSION THREAD

**Discussion by sebastiantiesmeyer** [BayraktarLab/cell2location#34] [Q&A] [answered]
Hi,
thanks for your wonderful tool!
I want to apply it to visium data at the moment. In the first step, your model runs a regression over single cell expression data to determine representative cell type signatures. 
Can you tell me where these signatures are stored in the final model? I wanted to access them for QC, but notoriously fail to locate them in the AnnData object.
Thanks a lot again!
--Sebastian

**Comment by vitkl** [comment, marked as answer]
Hi @sebastiantiesmeyer 

you can find all estimated parameters of the model in `adata.uns['regression_mod']['post_sample_means ']` and corresponding variable, cell, cell label names in `adata.uns['regression_mod']['var_names ']`, `'obs_names'`, `'fact_names'`. 

What kind of QC were you thinking about?
